types/react-instantsearch/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for react-instantsearch 5.2
+// Project: https://community.algolia.com/react-instantsearch/
+// Definitions by: Gordon Burgett <https://github.com/gburgett>
+//                 Justin Powell <https://github.com/jpowell>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.9
+
+export {};
```

Comment:
This needs to match package/index.js  typing where in index.js  `createConnector` is exported.